poet/poet.py

```diff
@@ -170,7 +170,7 @@ def main():
         for i, package in enumerate(args.single):
             data = research_package(package)
             print(RESOURCE_TEMPLATE.render(resource=data))
-            if i != len(args.single)-1:
+            if i != len(args.single) - 1:
```

Comment:
This isn't suggested by PEP-8 and I think it conveys better information about operator precedence as-is